XamlControlsGallery/Controls/ControlExample.xaml.cs

```diff
@@ -1,577 +1,417 @@
+        }
+
+        public static readonly DependencyProperty CSharpSourceProperty = DependencyProperty.Register("CSharpSource", typeof(object), typeof(ControlExample), new PropertyMetadata(null));
+        public Uri CSharpSource
+        {
+            get { return (Uri)GetValue(CSharpSourceProperty); }
+            set { SetValue(CSharpSourceProperty, value); }
+        }
+
+        public static readonly DependencyProperty SubstitutionsProperty = DependencyProperty.Register("Substitutions", typeof(IList<ControlExampleSubstitution>), typeof(ControlExample), new PropertyMetadata(null));
+        public IList<ControlExampleSubstitution> Substitutions
+        {
+            get { return (IList<ControlExampleSubstitution>)GetValue(SubstitutionsProperty); }
+            set { SetValue(SubstitutionsProperty, value); }
+        }
+
+        public static readonly DependencyProperty ExampleHeightProperty = DependencyProperty.Register("ExampleHeight", typeof(GridLength), typeof(ControlExample), new PropertyMetadata(new GridLength(1, GridUnitType.Star)));
+        public GridLength ExampleHeight
+        {
+            get { return (GridLength)GetValue(ExampleHeightProperty); }
+            set { SetValue(ExampleHeightProperty, value); }
+        }
+
+        public static readonly DependencyProperty WebViewHeightProperty = DependencyProperty.Register("WebViewHeight", typeof(int), typeof(ControlExample), new PropertyMetadata(400));
+        public int WebViewHeight
+        {
+            get { return (int)GetValue(WebViewHeightProperty); }
+            set { SetValue(WebViewHeightProperty, value); }
+        }
+
+        public static readonly DependencyProperty WebViewWidthProperty = DependencyProperty.Register("WebViewWidth", typeof(int), typeof(ControlExample), new PropertyMetadata(800));
+        public int WebViewWidth
+        {
+            get { return (int)GetValue(WebViewWidthProperty); }
+            set { SetValue(WebViewWidthProperty, value); }
+        }
+
+        public new static readonly DependencyProperty HorizontalContentAlignmentProperty = DependencyProperty.Register("HorizontalContentAlignment", typeof(HorizontalAlignment), typeof(ControlExample), new PropertyMetadata(HorizontalAlignment.Left));
+        public new HorizontalAlignment HorizontalContentAlignment
+        {
+            get { return (HorizontalAlignment)GetValue(HorizontalContentAlignmentProperty); }
+            set { SetValue(HorizontalContentAlignmentProperty, value); }
+        }
+
+        public static readonly DependencyProperty MinimumUniversalAPIContractProperty = DependencyProperty.Register("MinimumUniversalAPIContract", typeof(int), typeof(ControlExample), new PropertyMetadata(null));
+        public int MinimumUniversalAPIContract
+        {
+            get { return (int)GetValue(MinimumUniversalAPIContractProperty); }
+            set { SetValue(MinimumUniversalAPIContractProperty, value); }
+        }
+
```

Comment:
Main change is from 190 to 209. Commenting these lines to highlight diff since Github isn't highlighting them